devices/schneider_electric.js

```diff
@@ -981,4 +982,28 @@ module.exports = [
                 .withUnit('A').withDescription('Instantaneous measured electrical current on phase C'),
         ],
     },
+    {
+        zigbeeModel: ['W599001'],
+        model: 'W599001',
+        vendor: 'Schneider Electric',
+        description: 'Wiser smoke alarm',
+        fromZigbee: [fz.temperature, fz.battery, fz.ias_enroll, fz.ias_smoke_alarm_1],
+        toZigbee: [],
+        ota: ota.zigbeeOTA, // local OTA updates are untested
+        exposes: [e.smoke(), e.battery_low(), e.tamper(), e.battery(), e.battery_voltage(),
+            // the temperature readings are unreliable and may need more investigation.
```

Comment:
I saw you mail, what do you mean with:

> Anyway, temperature readings seem to work sporadically.

It may help to increase the reporting frequency (in `configure`), try with the code below, it should make it report every 5 seconds:

```
await reporting.temperature(endpoint, {min: 1, max: 5, change: 0});
```